### PR TITLE
Recipe Lottie Library

### PR DESCRIPTION
**Pull request by bdecorneliis**
# Summary
Adding Lottie library, guide to use and examples.

# Review
<div >

</div>

# Usages
###  Attributes to use in component LottieAnimationView in xml file:
 
 * lottie_loop - A boolean to ensure that the animation stays in a loop.
 * lottie_speed - This determines the speed of the animation.
 * lottie_autoplay - A boolean that starts the animation.
 * lottie_url - The JSON URL from the website.
 * lottie_rawRes - This contains the bundled animation on your app.
 * lottie_fileName - If you have added the JSON files to the assets folder, use this instead of lottie_rawRes.
 
 ### Resources:
  - Download the required JSON files or url from https://lottiefiles.com/
  - Save them to the assets folder of the project

### CODE REVIEW

**Review by rodrigo103 — DISMISSED** [review]
Looks good to me, well done!!

File: app/build.gradle [hunk 1]
```diff
@@ -145,6 +145,7 @@ buildscript {
     ext.firebase_version = '17.2.3'
     ext.crashlytics_version = '2.10.1'
     ext.mercadopago_version = '4.40.1'
+    ext.lottie_version = '3.4.0'
```
Comment (ffasitella) [inline app/build.gradle:148]: Is there a version conflict with the [last one](https://github.com/airbnb/lottie-android/releases/tag/v4.1.0)?

**Review by ffasitella — CHANGES_REQUESTED** [review]
Amazing job! I left some feedback!

File: app/build.gradle (hunk 1 above)
Comment (bdecorneliis) [reply app/build.gradle:148]: Apparently, android studio suggests me that version

**Review by ffasitella — APPROVED** [review]
Sorry for the delay!
Great job! 🥇

**Pull request merged**